app/measurement/models.py

```diff
@@ -118,6 +118,7 @@ class Stat(models.TextChoices):
         AVERAGE = 'avg', _('Avg')
         MINIMUM = 'min', _('Min')
         MAXIMUM = 'max', _('Max')
+        RECENT = 'recent', _('Recent')
```

Comment:
This wouldn't do anything currently for monitors/alarms. Was it meant to be added for widgets, like in app/import/load/load_stattypes.py?

Reply:
oh jeez, good catch.  I wish I could say this was the first time I confused those two modles